### PR TITLE
Added docs on testing section

### PR DESCRIPTION
**Pull request by osundwajeff**
@timlinux  I have added the `Testing` section to the docs as you had suggested. Kindly check it out.

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 13 min

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]
Minor spelling changes and then changes to img names.

Nice work on updating the guide/index and the nav section correctly.

File: docs/src/developer/guide/testing.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+```
+
+### CI Github actions
+
+Use of [continuous integration and playwright](https://playwright.dev/docs/ci-intro).
+
+The CI  for this project is present in the directory `.github/workflows` in the `build-and-test.yml` file.
+
+The action builds and tests for and push or a pull request is made into the main repository.
+![push or pull request](./img/testing-ci-gh-actions.png)
```
[the remark this answers is not in the document]
Comment (osundwajeff) [reply docs/src/developer/guide/testing.md:70]: Alright. I'll correct those ones @Jeremy-Prior

File: docs/src/developer/guide/testing.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+```bash
+$ npm init playwright@latest
+```
+
+**Note:**
+
+- By default it uses TypeScript(`*.ts`).
+
+This command will install all the required browsers and other dependencies. The [directory stucture](https://playwright.dev/docs/intro#whats-installed) will be as follows:
```
[the remark this answers is not in the document]
Comment (osundwajeff) [reply docs/src/developer/guide/testing.md]: Done.

File: docs/src/developer/guide/index.md [hunk 3]
```diff
@@ -10,3 +10,4 @@ In this section of the documentation, we aim to onboard developers onto the proc
 * **[Configuration:](./configuration.md)** This covers standard configuration tasks that need to be done once the site is brought online.
 * **[Design:](./design.md)** This section describes the visual design approach for the user interface.
 * **[Workflows:](./workflows.md)** Examples of typical workflows for adding features, fixing bugs etc.
+* **[Testing:](./testing.md)** This section focuses on setting up end to end testing for the project.
```
[the remark this answers is not in the document]
Comment (osundwajeff) [reply docs/src/developer/guide/index.md]: Done.

File: docs/src/developer/guide/testing.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (osundwajeff) [reply docs/src/developer/guide/testing.md]: Done.

**Review by Jeremy-Prior — APPROVED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 29 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 45 sec

**Pull request merged**